.github/workflows/workflow.yml

```diff
@@ -42,9 +42,16 @@ jobs:
           opam exec -- dune runtest zmq-lwt
 
       - name: zmq-async
-        if: ${{ matrix.ocaml-version != 'ocaml-base-compiler.5.0.0~beta1' }}
         run: |
           opam pin add zmq-async.dev . --no-action
           opam install --deps-only --with-doc --with-test zmq-async
           opam exec -- dune build zmq-async
           opam exec -- dune runtest zmq-async
+
+      - name: zmq-eio
+        if: ${{ matrix.ocaml-version == '5.0.0' }}
```

Comment:
I would maybe set this to `>=` for if the matrix adds 5.1.0.

Reply:
Good suggestion. I will update the code.

Reply:
Ahh. Bit too quick there :-) `>=` will do lexicographical comparison on strings, not test semantic versions. This means that `5.0.10` <= `5.0.9`. I will try to change the test matrix to make it less error prone.